lib/form526_backup_submission/configuration.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'common/client/configuration/rest'
+
+module Form526BackupSubmission
+  class Configuration < Common::Client::Configuration::REST
+    self.read_timeout = Settings.caseflow.timeout || 20 # using the same timeout as lighthouse
+
+    ##
+    # @return [String] Base path
+    #
+    def base_path
+      Settings.form526_backup.url
+    end
+
+    ##
+    # @return [String] Service name to use in breakers and metrics.
+    #
+    def service_name
+      'Form526BackupSubmission'
+    end
+
+    ##
+    # @return [Hash] The basic headers required for any Lighthouse API call
+    #
+    def self.base_request_headers
+      super.merge('apikey' => Settings.form526_backup.api_key)
+    end
+
+    ##
+    # Creates the a connection with parsing json and adding breakers functionality.
```

Comment:
Small grammar thing: "Creates the a" should be "Creates the" or "Creates a".

Reply:
This was copy/pasta'd from another class, I have fixed in 95e09b829eca560cc76df0c01083b9c01ba46af2